packages/p2p-v2/src/components/BuySellForm/BuySellForm.tsx

```diff
@@ -240,7 +240,7 @@ const BuySellForm = ({
                 )}
                 {isBuy && (
                     <>
-                        <Divider />
+                        <Divider color='#f2f3f4' />
```

Comment:
This has been duplicated many times, can we make this the default color for the divider?

Reply:
sure 👍